web/models/setting/setting.ex

```diff
@@ -0,0 +1,37 @@
+defmodule CoursePlanner.Setting do
+  @moduledoc """
+  This module holds the model for the setting table
+  """
+  use CoursePlanner.Web, :model
+
+  schema "settings" do
+    field :notification_frequency, :integer
```

Comment:
I got it, maybe you can add a comment in `@moduledoc` or even a tooltip in the view explaining the functionality.